<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>948</width>
    <height>683</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">basalt_gui</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QGroupBox" name="avaliable">
      <property name="title">
       <string>Avaliable</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QListView" name="avaliableList">
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="use">
           <property name="toolTip">
            <string>Use selected effect</string>
           </property>
           <property name="text">
            <string notr="true">-&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="edit">
           <property name="toolTip">
            <string>Open selected effect`s parameters</string>
           </property>
           <property name="text">
            <string>parameters</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="add">
           <property name="toolTip">
            <string>Add effect</string>
           </property>
           <property name="text">
            <string notr="true">+</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="addFromDir">
           <property name="toolTip">
            <string>Add all effects from directory</string>
           </property>
           <property name="text">
            <string>add from dir</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="deleteAvaliable">
           <property name="toolTip">
            <string>Remove selected effect from list</string>
           </property>
           <property name="text">
            <string notr="true">-</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="used">
      <property name="title">
       <string>Used</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="up">
           <property name="toolTip">
            <string>Move selected effect upper</string>
           </property>
           <property name="text">
            <string>up</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="down">
           <property name="toolTip">
            <string>Move selected effect down</string>
           </property>
           <property name="text">
            <string>down</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="deleteUsed">
           <property name="toolTip">
            <string>Remove selected effect from used effects list</string>
           </property>
           <property name="text">
            <string notr="true">-</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListView" name="usedList">
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>948</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionNew"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="actionAutoSave"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionNew"/>
   <addaction name="separator"/>
   <addaction name="actionSave"/>
   <addaction name="actionSaveAs"/>
   <addaction name="separator"/>
   <addaction name="actionSettings"/>
   <addaction name="separator"/>
   <addaction name="actionAutoSave"/>
  </widget>
  <action name="actionSave">
   <property name="icon">
    <iconset theme="document-save">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save file</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset theme="document-save-as">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Save as</string>
   </property>
   <property name="toolTip">
    <string>Save file as</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset theme="document-open">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open file</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+O</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset theme="preferences-system">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="toolTip">
    <string>Open settings</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset theme="document-new">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="toolTip">
    <string>Create new file</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+N</string>
   </property>
  </action>
  <action name="actionAutoSave">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Auto save</string>
   </property>
   <property name="toolTip">
    <string>Auto save file every 2 seconds</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
